### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0tcnTpXMxZ9jiBx2AlvOKYvRtbytjEUtnb6ftBLKMtBCoNO6-RcuiHk0M_aem_AYzctyGcvP2Uln2yPgaZe4PLSBlM-A58CVR9CaVuHzDHNWk6ooKZewj2GHcWH28XujU

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#12] [Comments]
# https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0tcnTpXMxZ9jiBx2AlvOKYvRtbytjEUtnb6ftBLKMtBCoNO6-RcuiHk0M_aem_AYzctyGcvP2Uln2yPgaZe4PLSBlM-A58CVR9CaVuHzDHNWk6ooKZewj2GHcWH28XujU

Day 3 - Beautiful Teamwork miles today: 93 total miles: 169 km today: 150 total km: 192

https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0tcnTpXMxZ9jiBx2AlvOKYvRtbytjEUtnb6ftBLKMtBCoNO6-RcuiHk0M_aem_AYzctyGcvP2Uln2yPgaZe4PLSBlM-A58CVR9CaVuHzDHNWk6ooKZewj2GHcWH28XujU

**Comment by keikomiller** [comment]
I wish I had thought of the CAN$ issue as well b/c I have some misc Canadian dollars from past trips…not enough to exchange, so it’s just sitting in the lockbox in an envelope.  I could have TOTALLY given to you, if any of us had thought of it.  Doh.
